ignite/services/network/testutil/suite.go

```diff
@@ -0,0 +1,39 @@
+package testutil
+
+import (
+	"testing"
+
+	"github.com/ignite-hq/cli/ignite/pkg/cosmosaccount"
+	"github.com/ignite-hq/cli/ignite/services/network/mocks"
+)
+
+type Suite struct {
```

Comment:
Can we please add detailed comments to each public struct/funcs here in this file and other testutil files?